### PR TITLE
newfeature commit

### PR DESCRIPTION
**Pull request by natehou**
add a testing message on the main page

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
     <canvas class="webgl"></canvas>
+    <h1>This is just a testing message!</h1>
```
Comment (victorhwn7255) [inline src/index.html]: Could you change the line please?

Comment (natehou) [reply src/index.html]: Sure!

Comment (victorhwn7255) [reply src/index.html]: Awesome, thanks Nate!

Comment (victorhwn7255) [inline src/index.html]: Change the headline

**Review by victorhwn7255 — CHANGES_REQUESTED** [review]

**Review by victorhwn7255 — APPROVED** [review]

**Pull request merged**